libs/api/src/main/java/de/cyface/api/PauseAndResumeStrategy.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Couldn't we move this back to the data-collector. Is it required anywhere else?

Reply:
The `libs/api/Authorizer` has to know if it needs to pause/resume requests, depending on what the API endpoints do (stream data or parse all before continuing).

Thus, it has to stay in `libs/apis`.